src/modules/call/ui/components/call-ended.tsx

```
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
+
+
+export const CallEnded = ({ }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)